lib/tasks/hybrid_cloud.rake

```diff
@@ -22,9 +22,9 @@ namespace :rh_cloud do |args|
       exit(1)
     end
     
-    organization = Organization.find_by(id: ENV['org_id'].to_i) # saw this coming in as a string, so making sure it gets passed as an integer.
-    
-    @uid = cp_owner_id(organization)
+    @organization = Organization.find_by(id: ENV['org_id'].to_i) # saw this coming in as a string, so making sure it gets passed as an integer.
+    @uid = cp_owner_id(@organization)
+    @hostname = ForemanRhCloud.foreman_host_name
```

Comment:
This is not an actual hostname. It could be the name of a cloned system. Is it OK?

Reply:
I was trying to use the function I added in my other pr, but I was getting an error. I can try to look at it further.